app/services/fact_parser.rb

```diff
@@ -125,6 +125,11 @@ def cores
   def disks_total
   end
 
+  # Kernel version
+  def kernel_version
+    facts['uname.release']
```

Comment:
I'd say the abstract fact parser shouldn't be implementing this.
```suggestion
```

Maybe you're looking for https://github.com/theforeman/foreman/blob/develop/app/services/katello/rhsm_fact_parser.rb to implement it for RHSM?

Reply:
Good point, I was incorrect. Will add it there and keep this:

```ruby
def kernel_version
end
```